enhancements/eus-to-eus-upgrade-impediment-reporting-capability.md

```diff
@@ -0,0 +1,209 @@
+---
+title: eus-to-eus-upgrade-impediment-reporting-capability
+
+authors:
+  - "@bentito"
+reviewers:
+  - TBD
+  - "@camilamacedo86"
+  - "@gallettilance"
+approvers:
+  - TBD
+creation-date: 2022-05-17
+last-updated: 2022-05-17
+status: provisional
+see-also:
+  - "/enhancements/none.md"
+replaces:
+  - "/enhancements/none.md"
+superseded-by:
+  - "/enhancements/none.md"
+---
+
+# eus-to-eus-upgrade-impediment-reporting-capability
+
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+A cluster upgrade between two versions (for instance from one EUS version to
+a later EUS version) may be hampered by critical, operator-delivered, services
+not being able to be upgraded.
+
+## Motivation
+
+Customers using X and Y and Z Red Hat Operators need to know that if they are using
+specific versions of each operator starting at certain OCP version, can they do a 
+successful EUS to EUS upgrade with no manual steps.
+
+
+### Goals
+
+A report showing impediments we want to determine for each operator and display:
+
+1. [Green] Can the operator remain the same version while OCP upgrades around it
```

Comment:
Presumably this means that the channel won't change, and you'll be looking at the metadata for either the currently installed CSV (if subscriptions are manual) or the head of channel?